scripts/webframeworks-deploy-tests/README.md

```diff
@@ -0,0 +1,19 @@
+# WebFrameworks Deploy Integration Test
+
+Function deploy integration test cycles through "create -> update -> update -> ..." phases to make sure all supported function triggers are deployed with correct configuration values.
```

Comment:
Does this test actually do this? Looks like we just deploy, right?

Reply:
No it doesn't, I updated the README, thank you for catching!! :)